tsconfig.json

```diff
@@ -1,52 +1,54 @@
 {
+  // until https://github.com/TypeStrong/ts-node/pull/1958 is released
+  // "extends": ["@tsconfig/recommended/tsconfig.json", "@tsconfig/node16/tsconfig.json"],
   "compilerOptions": {
-    "outDir": "dist",
-    "baseUrl": ".",
-
-    "target": "es2021",
-    "module": "esnext",
-    "moduleResolution": "node",
-    "lib": ["esnext"],
-    "allowSyntheticDefaultImports": true,
+    // part of @tsconfig/recommended
     "esModuleInterop": true,
-    "importHelpers": true,
-    "resolveJsonModule": true,
-    "sourceMap": true,
-    "declaration": true,
-
-    "noImplicitAny": true,
-    "noImplicitThis": true,
-    "alwaysStrict": true,
-    "noImplicitReturns": true,
-    "noUnusedLocals": true,
+    "forceConsistentCasingInFileNames": true,
```

Comment:
Isn't this more an eslint thing to check ? How is it related to types ?

Reply:
It makes sure case-sensitive and case-insensitive filesystems work interdependently. It makes the compiler differentiate `import './someThing.ts'` vs `import './SomeThing.ts'`.